src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -1,12 +1,12 @@
 package core.basesyntax;
 
 public class BinaryString {
-
-    /**
-     * Напишите метод toBinaryString(), который принимает на вход целое число value,
-     * а возвращает String с представлением этого числа в двоичном виде.
-     */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder result = new StringBuilder();
+        result.append(value == 0 ? "0" : "");
+        for (int i = value; i > 0; i /= 2) {
+            result.append(i % 2 != 0 ? "1" : "0");
```

Comment:
do we need '?' here? https://www.youtube.com/watch?v=P-UmyrbGjwE&list=PL7FuXFaDeEX1smwnp-9ri8DBpgdo7Msu2